python/lsst/faro/base/CatalogMeasurementBase.py

```diff
@@ -99,6 +99,10 @@ def __init__(self, config, *args, **kwargs):
         self.makeSubtask("measure")
 
     def run(self, **kwargs):
+        if 'shelveName' in self.measure.config.keys():
```

Comment:
Since you have a default value, isn't this test redundant?

Reply:
Currently, not all of the metric configs are inheriting from the recently create base metric config. We are in the process of making this conversion across all the metrics. For now, this line protects against the case that a given metric config is not inheriting from the base metric config.